### PR TITLE
doc: software_maturity: add ecosystem category and google fast pair

### PR DESCRIPTION
**Pull request by kapi-no**

### CODE REVIEW

**Comment by kapi-no** [comment]
You can wait with the review part until I remove the "Work in progress" label from the PR description

**Comment by kapi-no** [comment]
I decided to document the feature support in the follow-up change. The most important SW maturity documentation is for the given Fast Pair use case (the most high-level SW maturity documentation).

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 1]
```diff
@@ -229,6 +229,45 @@ The following table indicates the software maturity levels of the support for ea
...
+*****************
+
+The following sections contain the tables indicating the software maturity levels of the support for the following ecosystems:
+
+* Google Fast Pair
+
+Google Fast Pair
+================
+
+The following table indicates the software maturity levels of the support for each Google Fast Pair use case:
```
Comment (MarekPieta) [inline doc/nrf/releases_and_maturity/software_maturity.rst:242]: `for Google Fast Pair use cases integrated in the NCS` (e.g. we do not integrate audio use-cases at all)

**Review by MarekPieta — APPROVED** [review]

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 2]
```diff
@@ -229,6 +229,45 @@ The following table indicates the software maturity levels of the support for ea
...
+
+.. _software_maturity_fast_pair_use_case:
+.. list-table:: Google Fast Pair use case support
+   :header-rows: 1
+   :align: center
+   :widths: auto
```
Comment (annwoj) [inline doc/nrf/releases_and_maturity/software_maturity.rst:248]:
```suggestion
   :widths: auto
   
```

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 3]
```diff
@@ -229,6 +229,47 @@ The following table indicates the software maturity levels of the support for ea
...
+     - Experimental
+     - Experimental
+   * - Locator tag
+     - :ref:`fast_pair_locator_tag`
+     - Experimental
+     - Experimental
+     - Experimental
+     - Experimental
+     - Experimental
+
```
Comment (annwoj) [inline doc/nrf/releases_and_maturity/software_maturity.rst:272]:
```suggestion
.. toggle::

   The following table indicates the software maturity levels of the support for Google Fast Pair use cases integrated in the |NCS|:

   .. _software_maturity_fast_pair_use_case:

   .. list-table:: Google Fast Pair use case support
      :header-rows: 1
      :align: center
      :widths: auto

      * - Use case
      - |NCS| sample demonstration
      - nRF52832
      - nRF52833
      - nRF52840
      - nRF5340
      - nRF54L15
      * - Input device
      - :ref:`fast_pair_input_device`
      - Experimental
      - Experimental
      - Experimental
      - Experimental
      - Experimental
      * - Locator tag
      - :ref:`fast_pair_locator_tag`
      - Experimental
      - Experimental
      - Experimental
      - Experimental
      - Experimental
```

Just noticed after it rendered - we should include toggle button similarly to other items

**Review by MarekPieta — APPROVED** [review]

**Review by annwoj — APPROVED** [review]

**Review by divipillai — APPROVED** [review]

**Comment by kapi-no** [comment]
@anangl, this PR is ready to merge

**Pull request merged**